easybuild/easyconfigs/a/Autoconf/Autoconf-2.69-intel-2014b.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'Autoconf'
+version = '2.69'
+
+homepage = 'http://www.gnu.org/software/autoconf/'
+description = """Autoconf is an extensible package of M4 macros that produce shell scripts to automatically
+ configure software source code packages. These scripts can adapt the packages to many kinds of UNIX-like
+ systems without manual user intervention. Autoconf creates a configuration script for a package from a
+ template file that lists the operating system features that the package can use, in the form of M4 macro calls."""
+
+toolchain = {'name': 'intel', 'version': '2014b'}
```

Comment:
Yes. As it would be sufficient for CMake, Bison, Flex for sure, and it most probably sufficient for nurses as well. Should I change all of those to GCC as well?